src/test/java/com/mongodb/hibernate/internal/service/StandardServiceRegistryScopedStateTests.java

```diff
@@ -41,29 +36,4 @@ void differentStandardServiceRegistriesHaveDifferentStates() {
             }
         }
     }
-
-    @Test
-    @DisplayName(
-            "MongoConfigurationContributor is called once per StandardServiceRegistry, different MongoConfigurator instances passed")
-    @SuppressWarnings("try")
-    void mongoConfigurationContributorInvocationsAndMongoConfiguratorInstances() {
```

Comment:
I moved this test to `MongoConfigurationContributorTests`, as it is about `MongoConfigurationContributor`.